conf.d/fluentd/in_dummy.conf

```diff
@@ -0,0 +1,54 @@
+
+# TODO: using in_dummy for smoking test.
+
+# <source>
+#     @type dummy
+
+#     dummy {"message": "my dummy message"}
+
+#     auto_increment_key index
+#     rate 1
+#     size 1
+#     tag dummy.source
+
+#     # @label @DEBUG
+# </source>
+
+# <match dummy.source>
+#   @type kafka2
+#   brokers kafka:9093
+
+#   <buffer topic>
+#     @type memory
+#     flush_interval 3s
+#   </buffer>
+
+#   <format>
+#     @type json
+#   </format>
+
+#   default_topic general-topic
+
+#   required_acks -1
+#   max_send_limit_bytes 10485760
+# </match>
+
+# <match dummy.source>
+#     @type stdout
+# </match>
+# <match dummy.source>
+#   @type forward
+#   send_timeout 10s
+
+#   <server>
+#     name e2e
+#     host localhost
+#     port 20000
+#   </server>
+#   <parse>
+#     @type json
+#   </parse>
+#   <secondary>
+#     @type stdout
+#   </secondary>
+# </match>
```

Comment:
Three match represent three parallel outputs, am I right?

Reply:
No, only one of the `match`es will be used(that's the first one.)
I commented them all here as reference for the TODO task.